tests/examples/epcisDocWithSingleEvent.jsonld

```diff
@@ -2,34 +2,36 @@
   "@context": [
     "https://gs1.github.io/EPCIS/epcis-context.jsonld",
     {
-      "example": "http://ns.example.com/epcis/"
+      "gs1": "https://gs1.org/voc/"
```

Comment:
probably we need neither example nor gs1 context in this example, but does not hurt to keep it